Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests/cypress/e2e/Core-journeys/voluntary-conversion.cy.js

```diff
@@ -1,17 +1,21 @@
 /// <reference types ='Cypress'/>
+import projectList from '../../pages/projectList'
+import voluntaryProjectTaskList from '../../pages/voluntaryProjectTaskList'
+
+var projectName = 'Voluntary Cypress Project'
 
 describe('Voluntary conversion', { tags: ['@dev', '@stage'] }, () => {
    beforeEach(() => {
       cy.callAcademisationApi('POST', `cypress-data/add-voluntary-project.cy`, "{}")
          .then(() => {
-            cy.login({ titleFilter: 'Cypress Project' })
-               .then(() => {
-                  cy.get('[id="school-name-0"]').click();                  
-               });
+          //  projectList.selectVoluntaryProject()
+          projectList.selectProject(projectName)
          });
    })
 
    it('TC01: Voluntary conversion journey ', () => {
     // Go wild Dan!
+   voluntaryProjectTaskList.voluntaryProjectElementsVisible()
```

Comment:
Really minor one but can you just fix the indentation on this and the other change above please?

Reply:
D'oh!  I don't have Prettier installed on my Win device.  I'll just fix that thanks :)